src/queryResult/queryResultWebviewPanelController.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import * as qr from "../sharedInterfaces/queryResult";
+// import * as Constants from "../constants/constants";
+import { randomUUID } from "crypto";
+import VscodeWrapper from "../controllers/vscodeWrapper";
+import { ReactWebviewPanelController } from "../controllers/reactWebviewPanelController";
+import { QueryResultWebviewController } from "./queryResultWebViewController";
+import { registerCommonRequestHandlers } from "./utils";
+
+export class QueryResultWebviewPanelController extends ReactWebviewPanelController<
+    qr.QueryResultWebviewState,
+    qr.QueryResultReducers
+> {
+    private _correlationId: string = randomUUID();
+
+    constructor(
+        context: vscode.ExtensionContext,
+        private _vscodeWrapper: VscodeWrapper,
+        private _viewColumn: vscode.ViewColumn,
+        private _uri: string,
+        title: string,
+        private _queryResultWebviewViewController: QueryResultWebviewController,
+    ) {
+        super(
+            context,
+            "queryResult",
+            {
+                resultSetSummaries: {},
+                messages: [],
+                tabStates: {
+                    resultPaneTab: qr.QueryResultPaneTabs.Messages,
+                },
+                executionPlanState: {},
+            },
+            {
+                title: title,
```

Comment:
This title should have "(Preview)" appended (for now)

Reply:
Unlike the `QUERY RESULT` tab in the panel area, this title has the name of the query tab, not sure if it makes sense to add `(Preview)` here
<img width="943" alt="image" src="https://github.com/user-attachments/assets/9e4bf8b3-3b2a-4c9a-addc-c608844045c5">

Reply:
IMO, I think it's still good because not only do we want users to know that they're using the new query pane and that it's still in preview, but _we_ also want to know when users file bugs and include screenshots which version they're using.  We could always ask them, but you don't always get reliable/accurate answers. ;)

Reply:
added